LoopKitUI/Views/SettingDescription.swift

```diff
@@ -8,36 +8,6 @@
 
 import SwiftUI
 
-public enum TherapySetting: Int {
```

Comment:
I find it a bit odd that this patch shows removal of TherapySetting -- it should have already been removed when updating from dev (where my change already moved it) ... am I missing something?

Reply:
I did merge in dev, so not sure what happened (this didn't even show up as a merge conflict..)